### DISCUSSION TITLE
Disable Auto Pause

### DISCUSSION THREAD

**Discussion by iturdzonyou** [wolveix/satisfactory-server#253] [Q&A] [answered]
So.. in the in-game server settings, I have to manually untick the "pause when no players are on" so that the server continues when everyones off. The problem i'm having here is my server will sometimes restart due to power outages and such. Due to that, that untick I did will reset itself back to ticked and pause whenever no ones in again. 

I checked the serversettings.ini and it says mAutoPause=False. I would assume this means the pausing of the server while no players are on is disabled. Yet, it's still happening? Is there another way?

![hrtedx](https://github.com/user-attachments/assets/5e74110e-5479-47f0-a230-2b421420f363)

**Comment by wolveix** [comment]
Could you provide your full Docker run command/Docker Compose file? From the `README`, we use the `AUTOPAUSE` environment variable to determine this value within the .ini file.

**Reply by iturdzonyou** [reply]
Sorry saw this really late, yeah I saw that as I was reading the readme for 1.0. I added that into it, thank you!

**Reply by wolveix** [reply, marked as answer]
My apologies @skhalid93, I completely forgot about this discussion alongside all of the 1.0 changes! The short version is that `AUTOPAUSE` is configured in-game now, not from an environment variable. If you change the state in-game, it will now persist (as long as you're running the latest version of the image). Thanks!

**Reply by OdinIversen** [reply]
This has not been fully resolved. I have changed the settings in-game, many times. However, each time the server resets the auto pause is re enabled. thus every time I log back on the next day, the setting has changed and the game has not been running.

**Reply by wolveix** [reply]
@OdinIversen this is a known bug with the game, not the Docker image

**Reply by OdinIversen** [reply]
Ok, thank you for the quick reply. Then we will just have to wait for the bug to hopefully be resolved?

**Reply by wolveix** [reply]
Yes :)

**Reply by AshleighTheCutie** [reply]
Note this is still broken - Docker or not - on 1.1/CL#433351.
EDIT: Unchecking the box only applies until server restart, at which point it's reset again. There is also no `serversettings.ini` file in the config directory.

**Reply by porisius** [reply]
To me, it looks like it is in the `GameUserSettings.ini` file, that is the settings for Auto-Pause Disabled and Network Quality Ultra. Maybe something to thing about.

```
[/Script/FactoryGame.FGGameUserSettings]
mIntValues=(("FG.DSAutoPause", 0),("FG.NetworkQuality", 3))
```

**Reply by AshleighTheCutie** [reply]
Yeah, that Auto Pause line exists and is set to 0 - however it is not disabled when the server starts back up.

**Reply by wolveix** [reply]
This sounds like a game issue rather than container issue. Either way, this discussion isn't the right place for it :) Feel free to open an issue if it is a container issue!
